docs/fs.md

```
@@ -78,6 +78,176 @@ In between tests it empties the `InMemory`, restoring the file system to a prist
+await git.clone({
+  fs,
+  http,
+  dir: '/',
+  url: 'https://github.com/example/repo',
+  singleBranch: true,
+  depth: 1,
+})
+```
+
+> **Note:** All data is ephemeral — it is lost when the Worker terminates.
+> For persistence across requests see [Option 3](#option-3-persisting-across-requests-cloudflare-durable-objects) below.
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)